examples/slider/slider-multithumb.html

```diff
@@ -0,0 +1,274 @@
+        <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/3">Related Issues</a></li>
+        <li><a href="../../#slidertwothumb">Design Pattern</a></li>
+      </ul>
+    </nav>
+    <main>
+      <h1>Horizontal Multi-Thumb Slider Example</h1>
+      <div class="advisement">
+        <p>      <p><strong>WARNING!</strong> Some users of touch-based assistive technologies may experience difficulty utilizing widgets that implement this slider pattern because the gestures their assistive technology provides for operating sliders may not yet generate the necessary output.  To change the slider value, touch-based assistive technologies need to respond to user gestures for incrementing and decrementing the value by synthesizing key events.  This is a new convention that may not be fully implemented by some assistive technologies.  Authors should fully test slider widgets using assistive technologies on devices where touch is a primary input mechanism before considering incorporation into production systems.</p>
+      </div>
+      <p>The below example section includes a slider for setting a price range that demonstrates the
+        <a href="../../#slidertwothumb">multi-thumb slider design pattern.</a>
+        Users set a price range by moving the arrows (thumbs).
+        The slider has two thumbs: one for the minimum price and one for the maximum price.
+      </p>
+      <p>Similar examples include: </p>
+      <ul>
+        <li>
+          <a href="slider-color-viewer.html">Horizontal Slider Examples</a>:
+          Demonstrates using three horizontally aligned single-thumb sliders to make a color viewer.
+        </li>
+        <li>
+          <a href="slider-thermostat.html">Slider Examples with aria-orientation and aria-valuetext</a>:
+          A thermostat control with two single-thumb sliders that demonstrate using aria-orientation and aria-valuetext.
+        </li>
+      </ul>
+      <section>
+        <div class="example-header">
+          <h2 id="ex_label">Example</h2>
+        </div>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div id="ex1">
+          <h3>Hotel Price Range (in <abbr title="United States Dollars">USD</abbr>)</h3>
+          <div class="slider-multithumb">
+            <svg role="none" class="slider-group"  width="360" height="60">
+              <g aria-hidden="true" class="rail">
+                <rect height="0" width="0"/>
+              </g>
+              <g aria-hidden="true" class="range">
+                <rect height="0" width="0"/>
+              </g>
+              <g class="minimum"
+                role="slider"
+                tabindex="0"
+                aria-valuemin="0"
+                aria-valuenow="100"
+                aria-valuemax="250"
+                aria-label="Hotel Minimum Price">
```

Comment:
I don't think the label is adequate. The visual label is Hotel Price Range in USD. The aria-label needs to include the currency too

Reply:
@jnurthen 
James I updated the label seems to read will with JAWS and NVDA.

Reply:
thanks - looks good